modules/rosa-create-objects.adoc

```diff
@@ -126,7 +126,7 @@ $ rosa create cluster --cluster-name=<cluster_name> [arguments]
 |The identifier of one or more additional security groups to use along with the default security groups that are used with the control plane nodes created alongside the cluster. For more information on additional security groups, see the requirements for _Security groups_ under _Additional resources_.
 
 a|--cluster-name <cluster_name>
-|Required. The name of the cluster. When used with the `create cluster` command, this argument is used to set the cluster name and to generate a sub-domain for your cluster on `openshiftapps.com`. The value for this argument must be unique within your organization.
+|Required. The name of the cluster. When used with the `create cluster` command, this argument is used to set the cluster name and to generate a subdomain for your cluster on `openshiftapps.com`. The value for this argument must be unique within your organization.
```

Comment:
The cluster name isn't used to generate the subdomain, only the domain prefix is used for such purposes.

Reply:
What about cluster names that are less than 15 characters? Does the cluster name not generate the subdomain then? Thanks.

Reply:
For cluster names that are <= 15 chars, the name maybe be used to set the 'domain-prefix' value only if it not already used already. 

In all the cases, cluster name is a user friendly unique identifier for the cluster. It may hold up to 54 chars.
The domain-prefix is what really is used to generate the subdomain of the cluster's domain.

Reply:
Thanks. I'll update that. I am having asciidoc build issues so the PR is not ready for review. I will ping you and Ciaran and add it to the Slack channel when it is ready.

Reply:
Thanks